gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GeneratedResClassSpec.kt

```diff
@@ -55,9 +55,8 @@ private fun ResourceType.requiresKeyName() =
     this in setOf(ResourceType.STRING, ResourceType.STRING_ARRAY, ResourceType.PLURAL_STRING)
```

Comment:
Please fix the failing CI before merging

Reply:
the error is not related to changes

Reply:
It is better to always restart the checks, to be sure. These tests were failed for the first time in this PR:
<img width="1098" alt="image" src="https://github.com/user-attachments/assets/3769d77e-7f94-49a4-aa34-5744dbea5266" />

Reply:
I was just talking about a good practice :).

The last build is the same stability now:
![image](https://github.com/user-attachments/assets/4a109fec-bc10-4213-a37e-e9d2861a71a5)